internal/init/init.go

```diff
@@ -132,24 +124,6 @@ func (c *Client) NewCommand() *cobra.Command {
 	return initCmd
 }
 
-func getFirstProjectID(s packngo.ProjectService, userOrg string) (string, error) {
```

Comment:
I think we could preserve this behavior of choosing the first (sometimes only) organization project as the default for metal-cli when none is defined in the user preferences. I believe `default_project_id` preference is set in the console and not all users will have on defined.